proto/p4/config/p4info.proto

```diff
@@ -83,6 +83,7 @@ message P4Ids {
     ACTION = 0x01;
     TABLE = 0x02;
     VALUE_SET = 0x03;
+    CONTROLLER_HEADER = 0x04;
```

Comment:
What in a P4 program would become CONTROLLER_HEADER?  Sorry, I may be behind on the thinking here.

Reply:
Header types annotated with `@controller_header` (this is referred to as `controller_packet_metadata` in the P4Info). This is not architecture-specific IMO, so I added it next to the P4 built-in objects.